modules/fbx/fbx_parser/LICENSE.lic

```diff
@@ -1,6 +1,6 @@
 Open Asset Import Library (assimp)
```

Comment:
Same question about this.

If you want to make it clear that it's the original assimp stuff and not the license/credits for every single line of code, you could rename it `LICENSE.assimp.txt` and `CREDITS.assimp.txt` I guess.

Reply:
I did this differently just renamed back to CREDITS and LICENSE respectively